wp-content/themes/core/template-tags/titles.php

```diff
@@ -8,31 +8,38 @@
 /**
  * Sets up the proper title
  *
+ * @todo  Remove this function
```

Comment:
:100: I've got on my list to eliminate all the template-tag functions this go-round.

Reply:
nice